superset-frontend/src/explore/components/controls/DndColumnSelectControl/DndSelectLabel.tsx

```diff
@@ -66,7 +66,8 @@ export default function DndSelectLabel<T, O>(
         <ControlHeader {...props} />
       </HeaderContainer>
       <DndLabelsContainer canDrop={canDrop} isOver={isOver}>
-        {isEmpty(props.values) ? renderPlaceHolder() : props.valuesRenderer()}
+        {props.valuesRenderer()}
+        {displayGhostButton && renderPlaceHolder()}
```

Comment:
Nit: can we rename `renderPlaceHolder` to `renderGhostButton` just to be consistent?

Reply:
Done